cfspeedtest/cloudflare.py

```diff
@@ -191,9 +194,8 @@ def run_test(self, test: TestSpec) -> TestTimers:
                 test.type.value, url, data=data, timeout=self.timeout
             )
             coll.full.append(time.time() - start)
-            coll.server.append(
-                float(r.headers["Server-Timing"].split("=")[1].split(",")[0]) / 1e3
-            )
+            timing_match = TIMING_DURATION_RE.search(r.headers["Server-Timing"])
+            coll.server.append(float(timing_match.group(1)) / 1e3)
```

Comment:
In its current state, this change is fallible - much like before, but in a way that will crash instead of returning an erroneous value. We could handle this condition, but short of reporting `None` to invalidate the entire test, it would involve some compromise.